client/src/Components/Header.js

```diff
@@ -0,0 +1,144 @@
+                        </li>
+                        <li className="nav-item d-none d-sm-inline-block">
+                            <a href="#" className="nav-link">Contact</a>
+                        </li>
+                    </ul>
+                    {/* Right navbar links */}
+                    <ul className="navbar-nav ml-auto">
+                        {/* Navbar Search */}
+                        <li className="nav-item">
+                            <a className="nav-link" data-widget="navbar-search" href="#" role="button">
+                                <i className="fas fa-search" />
+                            </a>
+                            <div className="navbar-search-block">
+                                <form className="form-inline">
+                                    <div className="input-group input-group-sm">
+                                        <input className="form-control form-control-navbar" type="search" placeholder="Search" aria-label="Search" />
+                                        <div className="input-group-append">
+                                            <button className="btn btn-navbar" type="submit">
+                                                <i className="fas fa-search" />
+                                            </button>
+                                            <button className="btn btn-navbar" type="button" data-widget="navbar-search">
+                                                <i className="fas fa-times" />
+                                            </button>
+                                        </div>
+                                    </div>
+                                </form>
+                            </div>
+                        </li>
+                        {/* Messages Dropdown Menu */}
+                        <li className="nav-item dropdown">
+                            <a className="nav-link" data-toggle="dropdown" href="#">
+                                <i className="far fa-comments" />
+                                <span className="badge badge-danger navbar-badge">3</span>
+                            </a>
+                            <div className="dropdown-menu dropdown-menu-lg dropdown-menu-right">
+                                <a href="#" className="dropdown-item">
+                                    {/* Message Start */}
+                                    <div className="media">
+                                        <img src="dist/img/user1-128x128.jpg" alt="User Avatar" className="img-size-50 mr-3 img-circle" />
+                                        <div className="media-body">
+                                            <h3 className="dropdown-item-title">
+                                                Brad Diesel
+                                                <span className="float-right text-sm text-danger"><i className="fas fa-star" /></span>
```

Comment:
These hard coded values should come in from the controller (even if they're an sample template for now for now)

Instead of `Brad Diesel`, replace it with something like `{{this.userProfile.name}}`

Reply:
I don't have full context of these to know if it's actually coming from a profile interface; however it can be setup so that it can be scalable once we're pulling in the data :)